src/main/java/de/triology/cas/services/CesServicesManagerStageProductive.java

```diff
@@ -103,7 +104,18 @@ private void registerChangeListener() {
      * Creates and registers a new service for an given name
      */
     private void addNewService(String name) {
-        addNewService(name, "https://" + fqdn + "(:443)?/" + name + "(/.*)?");
+        String serviceId = "https://" + fqdn + "(:443)?/" + name + "(/.*)?";
+        if (registry instanceof RegistryEtcd){
+            try {
+                addNewService(name, serviceId, ((RegistryEtcd) registry).getCasLogoutUri(name));
+            } catch (GetCasLogoutUriException e) {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Either log or rethrow this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://ecosystem.cloudogu.com/sonar/coding_rules#rule_key=squid%3AS1166)